File: project_euler/js/euler_1.js
// Problem 1: Multiples of 3 and 5
function sumMultiples() {
  totalSum = 0;

  for (var i = 0; i < 1000; i++) {
    if ((i % 3 === 0) || (i % 5 === 0)) {
      totalSum += i;
    }
  }
  return totalSum;
}

File: project_euler/ruby/euler_1.rb
# Problem #1: Multiples of 3 and 5
def sum_multiples
  total_sum = 0

  (1...1000).each do |n|
    total_sum += n if (n % 3).zero? || (n % 5).zero?
  end

  puts "Sum of multiples of 3 or 5 below 1000 is #{total_sum}" # Output: 233168
end
